clients/unshaded/src/main/java/tachyon/client/file/options/SetStateOptions.java

```diff
@@ -73,4 +87,14 @@ private SetStateOptions(SetStateOptions.Builder builder) {
   public Boolean getPinned() {
     return mPinned;
   }
+
+  /**
+   * @return the ttl value, null if it is not set in builder; it specifies how long should the file
```

Comment:
let's be consistent with the description in `CreateOptions`: `the TTL (time to live) value; it identifies duration (in milliseconds) the created file should be kept around before it is automatically deleted`